lib/guess_html_encoding.rb

```diff
@@ -45,7 +45,7 @@ def self.encode(html, headers = nil)
     if html_copy.valid_encoding?
       html_copy
     else
-      html_copy.force_encoding('ASCII-8BIT').encode('UTF-8', :undef => :replace, :invalid => :replace)
```

Comment:
I think we should probably follow whatever logic most web browsers follow (as codified in the HTML spec), given that most people are probably using this library to parse webpages?